include/jwt-cpp/traits/danielaparker-jsoncons/traits.h

```diff
@@ -1,8 +1,10 @@
+#ifndef JWT_CPP_DANIELAPARKER_JSONCONS_TRAITS_H
```

Comment:
Is there a particular reason for adding header gaurds?

Reply:
header guard is missing only into jsoncons trait